### ISSUE TITLE
plot_gpytorch.py tutorial is broken with gpytorch v1.1.1

### ISSUE THREAD

**Issue by bcharlier** [getkeops/keops#64]
here is the error stack

```bash
Traceback (most recent call last):
  File "/home/bcharlier/keops/pykeops/tutorials/backends/plot_gpytorch.py", line 196, in <module>
    loss = -mll(output, train_y)
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/module.py", line 28, in __call__
    outputs = self.forward(*inputs, **kwargs)
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/mlls/exact_marginal_log_likelihood.py", line 51, in forward
    res = output.log_prob(target)
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/distributions/multivariate_normal.py", line 135, in log_prob
    inv_quad, logdet = covar.inv_quad_logdet(inv_quad_rhs=diff.unsqueeze(-1), logdet=True)
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/lazy/lazy_tensor.py", line 1033, in inv_quad_logdet
    args = self.representation()
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/lazy/lazy_tensor.py", line 1263, in representation
    representation += list(arg.representation())
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/lazy/lazy_evaluated_kernel_tensor.py", line 313, in representation
    return self.evaluate_kernel().representation()
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/utils/memoize.py", line 34, in g
    add_to_cache(self, cache_name, method(self, *args, **kwargs))
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/lazy/lazy_evaluated_kernel_tensor.py", line 280, in evaluate_kernel
    res = self.kernel(x1, x2, diag=False, last_dim_is_batch=self.last_dim_is_batch, **self.params)
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/kernels/kernel.py", line 396, in __call__
    res = lazify(super(Kernel, self).__call__(x1_, x2_, last_dim_is_batch=last_dim_is_batch, **params))
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/module.py", line 28, in __call__
    outputs = self.forward(*inputs, **kwargs)
  File "/home/bcharlier/.conda/envs/keops/lib/python3.7/site-packages/gpytorch/kernels/scale_kernel.py", line 92, in forward
    orig_output = self.base_kernel.forward(x1, x2, diag=diag, last_dim_is_batch=last_dim_is_batch, **params)
  File "/home/bcharlier/keops/pykeops/tutorials/backends/plot_gpytorch.py", line 124, in forward
    x_i, y_j = x1.div(self.lengthscale), x2.div(self.lengthscale)
TypeError: div(): argument 'other' (position 1) must be Tensor, not NoneType
```

**Comment by bcharlier** [comment]
up!

**Comment by jeanfeydy** [comment]
Fixed with e64cef1.
